src/flatc.cpp

```diff
@@ -241,6 +243,14 @@ int FlatCompiler::Compile(int argc, const char **argv) {
       } else if (arg == "--go-import") {
         if (++argi >= argc) Error("missing golang import" + arg, true);
         opts.go_import = argv[argi];
+      } else if (arg == "--go-nsimport-prefix") {
+        if (++argi >= argc)
+          Error("missing golang namespace import path prefix" + arg, true);
+        opts.go_nsimport_prefix = argv[argi];
+        if (!opts.go_nsimport_prefix.empty() &&
```

Comment:
use `flatbuffers::ConCatPathFileName(opts.go_nsimport_prefix, "")` like elsewhere in this file.